### ISSUE TITLE
Expanded icons

### ISSUE THREAD

**Issue by lucidobservor** [mephitrpg/rpg-cards#33]
Hello there. It seems that this tool gets all of its icons from game-icons.net. However, game-icons.net continues to add new icons, while this tool seems to use only a static list of them, which was probably current when you were writing it but which no longer is so. I've been making a lot of spell cards recently, and there have been over a dozen instances of finding the icon I wanted on game-icons.net, and then finding it isn't available in the card generator.

As I feature request, I'd like to suggest somehow getting the list of possible icons from game-icons.net, rather than from your static icons.js file. Though as a stopgap, you could simply update that file.

Please correct me if I've misunderstood something about how your tool works. It is otherwise fantastic.

**Comment by crobi** [comment]
Good point, they update their icons every couple of days. Even better would be if you could load icons from arbitrary URLs.

I've been experimenting with a different approach of generating the spell cards. The code is available in the `web-components` branch, and there's a (possibly slightly outdated) online [preview](http://crobi.github.io/rpg-cards/generator-next/generate.html).

In this new approach, the icons are not placed via a CSS class and a PNG background image, but instead via a `src` property of a custom tag. This loads a SVG icon directly from a URL and dynamically inserts it into the HTML. Currently, all icon paths are prefixed with `./icons/` in the generator, so you won't be able to load external icons, but you could try opening [this test page](http://crobi.github.io/rpg-cards/generator-next/components-test.html) and experiment with the icon URL.

Ultimatively, I want to use this new approach of loading icons, but am not sure how to make existing data files backward compatible yet enable external icon URLs at the same time.

**Comment by lucidobservor** [comment]
The preview definitely allows more icons than the currently live version. I
also like the formatting in the Contents section a lot more, so that's
excellent. This definitely serves my purposes, so thank you very much for
that!

I don't have any suggestions for the backwards compatibility, besides the
obvious: allow the importing of data from either format and call that
sufficient.
